infrastructure/identity-aware-proxy/iap_validating_server.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
Can you run this through gpylint for general formatting consistency?

Reply:
Done